bootstrap/pkg/kfapp/gcp/gcp.go

```diff
@@ -1961,26 +1961,14 @@ func (gcp *Gcp) Generate(resources kftypesv3.ResourceEnum) error {
 	}
 	// the runGetGCPCredentials don't seem to work because those are shelled out commands
 	// Added an alternate way to set using enironment variables
-	gcp.kfDef.Spec.Project = os.Getenv("PROJECT")
 	if gcp.kfDef.Spec.Project == "" {
-		return &kfapis.KfError{
-			Code:    int(kfapis.INVALID_ARGUMENT),
-			Message: "Project not specified.",
-		}
+		log.Warnf("GCP Project is not set, please set it in KFDef.")
```

Comment:
should we just log warning or should we error out?  same for all other fields.

Reply:
We can error out, I'll get a commit in.

Reply:
Done.